demisto_sdk/commands/generate_docs/common.py

```diff
@@ -146,13 +166,15 @@ def stringEscapeMD(st, minimal_escaping=False, escape_multiline=False):
     return st
 
 
-def execute_command(command_example):
+def execute_command(command_example, insecure: bool):
     errors = []
     context = {}
     md_example = ''
     cmd = command_example
     try:
         runner = Runner('')
+        if insecure:
+            runner.set_insecure(insecure)
```

Comment:
this is what i meant. no need for the set function.
```suggestion
        runner = Runner('', insecure=insecure)
```

init function of runner:
![image](https://user-images.githubusercontent.com/30797606/77097057-de539380-6a18-11ea-8de5-32122e8f2f0b.png)